### PR TITLE
Remove batch concept

### PR DESCRIPTION
**Pull request by amarkrsinha1997**
Removed the batch concept completely

### CODE REVIEW

File: src/controllers/assignments/assignment-controller.ts [hunk 1]
```diff
@@ -283,34 +290,68 @@ export default class AssignmentController {
...
+        'kanika17@navgurukul.org',
+        'rani17@navgurukul.org',
+        'khusboo17@navgurukul.org',
+      ];
+
+      database('users')
+          .select('users.email')
+          .where({'users.id': request.userId})
+          .then((rows)=> {
+            for(let i = 0; i < developerEmails.length; i++){
```
Comment (abhishekgupta92) [inline src/controllers/assignments/assignment-controller.ts]: developerEmails.indexOf(rows[0].email) > -1

is a more concise way to write this code

File: src/controllers/courses/course-routes.ts [hunk 2]
```diff
@@ -27,7 +26,7 @@ export default function (server: Hapi.Server, serverConfigs: IServerConfiguratio
                           3. All courses (includes courses from 1 and 2.)',
             response: {
                 schema: Joi.object({
-                    "facilitatingCourses": Joi.array().items(facilitatingCourseSchema),
+                    "facilitatingCourses": Joi.array().items(courseSchema),
```
Comment (abhishekgupta92) [inline src/controllers/courses/course-routes.ts]: change the code such that both the backend doesn't send facilitatingCourses doesn't send it, and frontend doesn't need it. 

ab you aren't facilitator for a course anymore, ,we are going for a different way of permissions altogether

File: src/seed-courses/database.ts [hunk 3]
```diff
@@ -41,6 +41,33 @@ let _generateExerciseAddOrUpdateQuery = function(exerciseInfo) {
     return query;
 };
 
+export const findFaciliator = function(email) {
```
Comment (abhishekgupta92) [inline src/seed-courses/database.ts]: findFaciliator 

ki spelling galat hai

**Review by abhishekgupta92 — CHANGES_REQUESTED** [review]

File: courseUpdate.sh [hunk 4]
```diff
@@ -5,7 +5,7 @@ echo "(Unless Error Message is attached here.)"
 
 #cd /home/galileo
 
-export GALILEO_ENV='prod'
+export GALILEO_ENV='dev'
```
Comment (abhishekgupta92) [inline courseUpdate.sh]: fix this

Comment (abhishekgupta92) [reply courseUpdate.sh]: also add SENTRY

**Review by abhishekgupta92 — CHANGES_REQUESTED** [review]

File: src/controllers/courses/course-routes.ts (hunk 2 above)
Comment (amarkrsinha1997) [reply src/controllers/courses/course-routes.ts]: Should I remove the facilitating courses concept also? and there should be only enrolled and available courses? @abhishekgupta92

**Comment by abhishekgupta92** [comment]
Yes, I agree

--
P.S. Global warming is not cool! Save paper, save earth :)

File: src/controllers/courses/course-routes.ts (hunk 2 above)
Comment (abhishekgupta92) [reply src/controllers/courses/course-routes.ts]: Yes, I agree

**Pull request merged**